mlrun/errors.py

```diff
@@ -3,7 +3,7 @@
 import requests
 
 
-class MLRunBaseError(Exception):
+class BaseError(Exception):
```

Comment:
Removing `MLRun` prefix is something I thought about suggesting in my review to the module however decided not to.

The reason is to make this also usable (without risk of shadowing / confusion) if users import this package "improperly" meaning
Not:
```py
import mlrun.errors
```
But, like `mlrun` often does with other modules, using:
```py
from mlrun.errors import HTTPError
```
For the second scenario I suggest to keep the prefix.
Even if mlrun is 100% in that regard, the 2nd usage is still python-legit, so in package where the names will often collide with other package like errors, I think it's preferable to keep this redundancy
The important thing is, IMO that all errors here will be consistent.

Reply:
Yeah I thought about it, 
It is of course easily solvable using `as` e.g.
```
from mlrun.errors import BaseError as MLRunBaseError
from other_package.errors import BaseError as OtherPackageBaseError
```
feels very redundant, and having both `MLRun` prefix and `Error` suffix to everything generates very long names, 
I'm against, but don't feel strongly about it, let's go for democracy, @quaark, what do you choose ?
of course I agree that which ever way we choose we need to be consistent
(at first I asked @quaark to do only the Base one with `MLRun` prefix since I thought it will be a must since `BaseException` is already a thing, but we went on `Error` so no problem)

Reply:
I personally agree with @omesser. I preferred an MLRun prefix from the start :)

Reply:
@Hedingber  - maybe I haven't made it clear, but the scenarios I put are not for how we should import it in mlrun, but valid USER code. the USER can import without `as` and you don't know / can't control it.